Assets/Scripts/GameNotificationsManager.cs

```diff
@@ -74,8 +74,10 @@ public enum OperatingMode
             QueueClearAndReschedule = Queue | ClearOnForegrounding | RescheduleAfterClearing,
         }
 
+#pragma warning disable CS0649
         [SerializeField, Tooltip("The operating mode for the notifications manager.")]
         private OperatingMode mode;
+#pragma warning restore CS0649
```

Comment:
Can we set a default value to it rather than disabling the warning?

Reply:
Fixed